desktop-widgets/locationinformation.cpp

```diff
@@ -223,8 +223,9 @@ void LocationInformationWidget::initFields(dive_site *ds)
 
 void LocationInformationWidget::on_GPSbutton_clicked()
 {
+	QFileInfo finfo(system_default_filename());
 	QString fileName = QFileDialog::getOpenFileName(this, tr("Select GPS file to open"),
-								tr("/home"),
+								finfo.absolutePath(),
```

Comment:
so this will default to the `username.xml` filename we use as default for the data file. I think that's a bit confusing.

Reply:
I have completed all updates that I can see.  Yes, this points to the default folder containing the dive log. This is the only remaining issue. What default directory to use in a platform-independent way?